test/cuda-arch/base/clang@10.0.1_nvcc_c++14_lassen.cmake

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
```

Comment:
When this gets added to the blt repo, we should move these host-configs to the host-config directory.